playbook/app/pb_kits/playbook/pb_card/_card_mixin.scss

```
@@ -45,6 +50,12 @@ $pb_card_padding:(
   transition-duration: $transition_short;
   transition-timing-function: $easeIn;
 
+  @each $name, $color in $background_colors {
+    &[class*=_#{$name}] {
```

I think we want to make this a bit more specific so we don't have un-intentional style bleed

```suggestion
    &[class*=background_#{$name}] {
```

---

Yes I did run into some style bleeding and just figured out how to avoid it but this looks a lot easier.